app/controllers/event.js

```diff
@@ -3,20 +3,31 @@ const HANDLER = require('../utils/response-helper')
 const HttpStatus = require('http-status-codes')
 const permission = require('../utils/permission')
 const helper = require('../utils/paginate')
+const notificationHelper = require('../utils/notif-helper')
+const notification = {
```

Comment:
this can be a class object

Reply:
As we are not using object-oriented design as of now so haven't implemented it as a class.
Should we create a single class for this and use the object instance of that?
As I am not sure about this.